test/unit/spec/EnrollChoices_spec.js

```diff
@@ -61,7 +64,7 @@ function (Okta, OktaAuth, Util, EnrollChoicesForm, Beacon, Expect, FactorUtil, R
       index:      8,
       factorType: 'web'
     },
-    'WINDOWS_HELLO': {
+    'WEBAUTHN': {
```

Comment:
we are not testing for windows_hello anymore ?

Reply:
This object is only used for factor cardinality tests. Not all factors are tested in this object - only u2f, okta verify, and now webauthn

Reply:
wouldnt it be good in here to check windows hello too ? 

https://github.com/okta/okta-signin-widget/blob/662a286b906e98a51152ee5d7bef8db2b5570ea4/test/unit/spec/EnrollChoices_spec.js#L430

Reply:
I actually think these tests are not factor specific, so that's why some may be left out. But added WINDOWS_HELLO for good measure.